.github/workflows/evm-benchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: EVM benchmark
```

Comment:
@minh-bq we can add some infor in the name here for easily tracking 

For example 
```
run-name: E2E Reuse Test ${{ inputs.rpc_endpoint }}, path ${{ inputs.test_path }} by ${{ github.actor }}
```

Reply:
What information that you think is needed here?

Reply:
old commit - new commit - author
For quickly checking if we triggering manual mode.